src/data/repositories/NotificationDefaultRepository.ts

```diff
@@ -47,6 +51,18 @@ export class NotificationDefaultRepository implements NotificationRepository {
         });
     }
 
+    private _delete(notifications: Notification[]): FutureData<void> {
+        return Future.fromPromise(
+            this.storageClient.removeObjectsInCollection(
+                notificationsDataStore,
+                notifications.map(notification => notification.id)
+            )
+        ).flatMapError(error => {
+            console.error(`Notification (remove): ${error}`);
+            return Future.error(i18n.t("An error has occurred deleting notifications"));
```

Comment:
The `error` object may contain useful information about what went wrong, we may include it also in the future error string.